src/dialogSettings.html

```diff
@@ -113,7 +113,17 @@
 
       function inputValidatyHandler() {
         var validaty = true;
-        input.forEach(function(el) { return validaty && (validaty = el.nodeName !== 'INPUT' || el.value.length > 0); });
+        input.forEach(function(el) {
+          switch(el.name) {
+            case 'jira_url':
+              validaty = validaty && (el.value.indexOf('https://') !== -1);
+            break;
```

Comment:
indent "break" so its clear its part of the line case statement block.

Reply:
good catch